src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    private double site;
```

Comment:
Consider renaming the `site` variable to `side` for clarity and consistency with the context of a square's side length.